backup.c

```diff
@@ -1128,8 +1129,8 @@ pg_start_backup(const char *label, bool smooth, pgBackup *backup)
 	params[1] = smooth ? "false" : "true";
 
 	/* 3rd argument is 'non-exclusive' (assumes PG version >= 9.6) */
```

Comment:
Do we need to change the comment?

Reply:
We need to check if there are other comments or documents we have to fix at the same time?

Reply:
I checked the document and  changed zhe comment.